bin/cli.js

```diff
@@ -27,13 +27,8 @@ function parsePort(port) {
 }
 
 function parseBoolean(bool) {
-  if (!bool) {
-    return false
-  }
-  if (bool === 'false' || bool === '0') {
-    return false
-  }
-  return Boolean(bool)
+  const nonBoolValue = !bool || bool === 'false' || bool === '0'
```

Comment:
The variable name is not clear. It is a bool value. And the check does not check if it is a non bool value.

The function as written before seems to me to be clearer than the refactor.